charts/benchmark/README.md

```diff
@@ -0,0 +1,137 @@
+# wasmCloud Benchmarking Chart
+
+This chart is primarily used for benchmarking wasmCloud deployments within a Kubernetes cluster. It
+can be used to test something outside of the cluster as well, but this is not a tested use case.
+This chart has a self contained stack of Grafana, Loki, Tempo, OTEL collectors, and Prometheus as
+well as installation for the [K6 Operator](https://github.com/grafana/k6-operator) for running the
+actual benchmark. This setup is meant to be temporary as many users are likely to have their own
+Grafana or OTEL stacks. This setup is purely for capturing data for the test. By default, this is a
+"push button" chart that will install the stack and run the benchmark for you. It can then be
+uninstalled so as to not take up resources.
+
+## Basic Usage
+
+A simple install command looks like below:
+
+```bash
+helm upgrade --install my-benchmark --version 0.1.0 oci://ghcr.io/wasmcloud/charts/benchmark --wait --set test.url=http://hello-world:8000
```

Comment:
Do we need a release pipeline for this? Suggest changing this to just be the local path for now if we want to do it in a follow up 😄

Reply:
Full local instructions (what I went through):

```bash
cd charts/benchmark
helm repo add https://grafana.github.io/helm-charts
helm repo add grafana https://grafana.github.io/helm-charts
helm repo add opentelemetry https://open-telemetry.github.io/opentelemetry-helm-charts
helm dependency build
helm upgrade --install my-benchmark --version 0.1.0 ./ --wait --set test.url=http://hello-world:8000
```

Reply:
All you should need for local is `helm dep up`. I forgot to add directions to README for that though.

As for release pipeline, see #3974

Reply:
Added instructions to the README. Feel free to resolve this if they look good